staging/kos/etcd-tls/cnf/etcd-client.cnf

```diff
@@ -0,0 +1,11 @@
+[req]
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+[req_distinguished_name]
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
+subjectAltName = @alt_names
+[alt_names]
```

Comment:
I thought we do not need any subjectAltNames for the client cert.  Do we?

Reply:
Currently is useless (as pointed out by your next comment), but we might want to restrict the names to the etcd-operator and network API server pods.

Reply:
I think the SANs in a client cert are not used for anything.

Reply:
I removed this SAN and it is working.